.github/workflows/provider-test.yml

```diff
@@ -60,4 +63,7 @@ jobs:
           ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
           ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
 
+      - name: Clean Up OIDC Token File Path
```

Comment:
at the moment this'll only run if it's successful, we can use an `always` for this to ensure it's always run: https://github.com/hashicorp/pandora/blob/6569798f7478abc0f7b23a120c13ac092e631fd3/.github/workflows/automation-regenerate-go-sdk.yaml#L57